packages/corelib/src/playout/infinites.ts

```diff
@@ -549,7 +549,6 @@ export function processAndPrunePieceInstanceTimings(
 	}
 
 	const groupedPieces = _.groupBy(
-		// TODOSYNC tv2 to write some tests and restore: keepDisabledPieces ? pieces.filter((p) => !(p.disabled && p.hidden)) : pieces.filter((p) => !p.disabled),
```

Comment:
Should the mentioned line be restored?

Reply:
I would say no, given that the variable name is called "keepDisabled..." so the filtering should only be affected by the disabled property.
I must admit that I don't know our system well enough to determine if it is important that the hidden attribute is also been taking into account here

Reply:
The issue that this was supposed to fix was as follows: When starting an overlay graphic adlib (e.g. a bund), the piece in the UI would show in full for a while, and then would either disappear or would be cut short, even though the graphic was still on air. It was due to _some_ hidden pieceInstance getting in a way of the actual pieceInstance with the graphic. I think this can be left as NRK has it, but needs to be verified during testing.